content/tagging/using_tags.md

```diff
@@ -12,105 +12,271 @@ further_reading:
   text: Learn how to assign tags
 ---
 
-After you have assigned tags at the host and [integration][1] level, you can start using them to filter and group in interesting ways. There are several places you can use tags:
+After [assigning tags][1], you can start using them to filter and group in interesting ways.
 
-- [Events List][2]
-- [Dashboards][3]
-- [Infrastructure List][4]
-- [Host Map][5]
-- [Monitors][6]
+## Events
 
-## Using tags in the Events List
+The [Events Stream][2] shows all the events that have occurred in your environment over the time period specified. Using tags to filter down this list is very beneficial. Enter `tags:` followed by a tag to see all the events that come from a host or [integration][3] with that tag. The example below is used to search for the tag `service:coffee-house` using `tags:service:coffee-house`. To search multiple tags, separate each tag by a comma - `tags:service:coffee-house,host:coffeehouseprod`.
```

Comment:
`Using tags to filter down this list is very beneficial` => this is vague
`Use tags to filter down the event list and focus on sub-set of events`

r/`that come from a host or [integration][3] `/`that come from a host or an [integration][3] `